inc/update.php

```diff
@@ -514,217 +500,6 @@ function largo_update_custom_less_variables() {
 	}
 }
 
-/**
- * Checks for use of deprecated widgets and posts an alert
- */
-function largo_check_deprecated_widgets() {
```

Comment:
@aschweigert should be all set now - let me know if you want me to rebase to clean up the commits